File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/orders/Currency.java

package com.artmurka.artmurkaapp.data.model.pojo.itemlist.orders;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Currency {

    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("rate")
    @Expose
    private String rate;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getRate() {
        return rate;
    }

    public void setRate(String rate) {
        this.rate = rate;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/orders/Currency.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.orders

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Currency {

    @SerializedName("id")
    @Expose
    var id: String? = null
    @SerializedName("rate")
    @Expose
    var rate: String? = null

}
